docs/developer/sdm-concepts.md

```diff
@@ -1,133 +1,17 @@
-between the SDM and API that is resilient to
-interruptions in connectivity.
-
-[ws]: https://en.wikipedia.org/wiki/WebSocket (WebSocket)
-
-### Events
-
-The heart of Atomist is its event handling. As your code flows from
-commit through to deployment and beyond, Atomist receives events,
-correlates the incoming data with its previous knowledge, and invokes
-your event handlers with rich context. This enables your automations
-to perform tasks such as:
-
-* Scanning code for security or quality issues on every push
-* Driving deployments and promotion between environments
-* Performing custom actions on deployment, such as kicking off
-    integration test suites.
-
-The Atomist correlated event model also enables Atomist to provide you
-with visibility throughout the commit to deployment flow, in Slack or
-through the Atomist web interface.  See [Events][events] for more
-information.
-
-[events]: event.md (Atomist - Events)
-
-Event handlers subscribe to events using [GraphQL][graphql]
-subscriptions against the Atomist cortex. The following GraphQL
-subscribes to completed builds, returning related data such as the
-last commit and any linked Slack channels:
-
-[graphql]: http://graphql.org (GraphQL)
-
-```graphql
-subscription OnBuildComplete {
-  Build {
-    buildId
-    buildUrl
-    compareUrl
-    name
-    status
-    commit {
-      sha
-      message
-      repo {
-        name
-        owner
-        gitHubId
-        allowRebaseMerge
-        channels {
-          name
-          id
-        }
-      }
-      statuses {
-        context
-        description
-        state
-        targetUrl
-      }
-    }
-  }
-}
-```
-
-When using TypeScript (our recommended language), an event handler can
-subscribe to such events with the benefit of strong typing. For
-example, this Atomist event handler can respond to the above GraphQL
-subscription:
-
-```typescript
-@EventHandler("Set status on build complete",
-    GraphQL.subscriptionFromFile("graphql/subscription/OnBuildComplete.graphql"))
-export class SetStatusOnBuildComplete implements HandleEvent<OnBuildComplete.Subscription> {
-
-    public async handle(event: EventFired<OnBuildComplete.Subscription>,
-    	ctx: HandlerContext,
-    	params: this): Promise<HandlerResult> {
-```
-
-This underlying GraphQL/event handler infrastructure is generic and
-powerful. However, many things are better done at a higher level. This
-project provides a framework above this infrastructure that makes
-typical tasks far easier, while not preventing you from breaking out
-into lower level functionality.
+The heart of Atomist is its event handling. We're going to take a look
+at some of the core concepts and functionality
+that demonstrates the strengths of an SDM over traditional CI/CD pipelines.
 
 ## SDM process lifecycle
 
-The SDM lifecycle will be familiar to those developing
-persistent applications.
+The SDM lifecycle ought to be familiar to the flow of any persistent
+application:
```

Comment:
The wording of this phrase trips me up. I _think_ we're trying to say that the SDM lifecycle should be familiar to anyone who has worked with persistent applications?